apps/hash-graph/postgres_migrations/V2__ontology_tables.sql

```diff
@@ -86,25 +86,37 @@ CREATE TABLE IF NOT EXISTS
   );
 
 CREATE TABLE IF NOT EXISTS
-  "property_type_property_type_references" (
+  "property_type_constrains_properties_on" (
```

Comment:
Should we be modifying the migrations in place? I thought we'd be writing new ones

Reply:
Until we have a persisted data store, we're using existing migration files.